doc/zh-cn/install-dependencies.md

```diff
@@ -54,7 +52,10 @@ sudo yum install yum-utils
 sudo yum-config-manager --add-repo https://openresty.org/package/centos/openresty.repo
 
 # 安装 OpenResty 和 编译工具
-sudo yum install -y openresty curl git gcc luarocks lua-devel
+sudo yum install -y openresty curl git gcc lua-devel openresty-openssl-devel
```

Comment:
can we use the `luajit` of openresty? if yes, we do not need to install the `lua-devel`.

Reply:
Updated.